### DISCUSSION TITLE
Tested on ....

### DISCUSSION THREAD

**Discussion by BeryBurnout** [coffeegreg/YTuner#23] [Show and tell]
works with Yamaha RX-V3900, server on RPi3.
Browsing is unusable because of speed but Bookmarks and My Stations work well

**Comment by coffeegreg** [comment]
Thank you for the information that YTuner works with your Yamaha RX-V3900.

> Browsing is unusable because of speed

I assume that this is about browsing `Radiobrowser` resources. What type of cache did you use? (option `RBCacheType` of `ytuner.ini` file) ?
[catDB,catMemDB,catPermMemDB] are best choice but require SQLite library. If you're not familiar with this lib you can read more about it [here](https://github.com/coffeegreg/YTuner/blob/master/doc/SQLITE.md). Please note that if YTuner does not detect your SQLite3 library correctly, it will switch to `catFile` cache mode but will inform you about it with a message `Changing cache to "catFile"`.
* [catDB,catMemDB,catPermMemDB] - cache of whole YTuner's usable radio-browser.info content. (The first download and processing of all data from radio-browser.info may take a few minutes but this is a one-time operation and its purpose is to build your local copy of the database which you will use until you decide to refresh it again)
* [catFile,catMemStr] - cache of previously browsed categories and stations in files/memory cache.

Test each option and choose the best one for you. 
> Tip: Each change of options values in the `ytuner.ini` file requires restarting YTuner.

For each of them, the delay should not be longer than a few or a dozen milliseconds. This makes browsing very convenient. 
Your RPi3 is at least several times more powerful than needed. Please, let us know which option will be best for you. :+1:

**Reply by BeryBurnout** [reply]
i have to thank for the nice piece of software, yes browsing with the Amps RC is for patient people :-) 
RBCacheType=catPermMemDB is in the ini unchanged. 
i will try to change the cachetype but for the moment i had to revert the dns settings since openweathermap stopped working on domoticz on the same rpi3 so i will use another DNS config

**Comment by Prideland** [comment]
works fine with Yamaha RX-A820

Thank you for the nice software

**Reply by coffeegreg** [reply]
Hi @Prideland 
Thank you for testing YTuner with your Yamaha RX-A820. I'll add it to the "Confirmed working" list.

**Comment by breml** [comment]
Works fine with Denon RCD-N7 running YTuner as Docker Container on my QNAP.
I even observed the startup time from pressing "Power On" until "Music is played" (from the last used internet radio) to be faster (might be because of using fixed IP instead of DHCP, but could also be related to quicker responses on the "check for updates" part during startup).

I have not tested the Bookmarks, since I am unsure if this is even supported on the Denon RCD-N7. I am under the impression, that "Favorites" are stored on the device it self and not on yTuner, but I might be wrong about this. Anyway, with the predefined list of stations, I have configured in the stations.ini I am more than happy.

Thank you for this nice software.

**Reply by coffeegreg** [reply]
Hi Lukas (@breml)
I'm glad to know that YTuner works fine with your Denon RCD-N7. :speaker: :notes: 
Thanks for letting us know. I'll add your Denon to the `Confirmed Working` list. :+1: 
>I even observed the startup time from pressing "Power On" until "Music is played" (from the last used internet radio) to be faster (might be because of using fixed IP instead of DHCP, but could also be related to quicker responses on the "check for updates" part during startup).

Faster than ... ? :smiley:  Original vTuner service ? :wink: Well ... From what I remember vTuner was never very fast when it was still working for everyone ... for free. :snail: 
If you do not want your Denon to regularly check for new firmware updates, use a sniffer (Wireshark or similar) to check what address your AVR connects to check firmware version and also add it to the `InterceptDNs` settings in the `[DNSServer]` section of the `ytuner.ini` file. I did this for my Yamahas. :smiley: 

> I have not tested the Bookmarks, since I am unsure if this is even supported on the Denon RCD-N7.

Yes, It does. :+1:  See user manual page 26. Bookmarks are great, when you're browsing `Radiobrowser` and find a great music station you'd like to memorize with a just simple click(s) of the remote control... or remove from it. :+1: 

> I am under the impression, that "Favorites" are stored on the device it self and not on yTuner, but I might be wrong about this.

"Favorites" are stored on your QNAP. :+1: YTuner is native app build with FPC and that's why is pretty fast. :rocket: 
It seems you are a great Go developer :clap:  so you know what I'm talking about. :wink: 

> Anyway, with the predefined list of stations, I have configured in the stations.ini I am more than happy.

Please try `Radiobrowser` resources too... with one of high efficient cache options (see `RBCacheType` settings in the `[Radiobrowser]` section of the `ytuner.ini` file. :+1: 

> Thank you for this nice software.

Stay tuned for next features... :wink:

**Reply by breml** [reply]
> Faster than ... ? 😃 Original vTuner service ? 😉 Well ... From what I remember vTuner was never very fast when it was still working for everyone ... for free. 🐌 If you do not want your Denon to regularly check for new firmware updates, use a sniffer (Wireshark or similar) to check what address your AVR connects to check firmware version and also add it to the `InterceptDNs` settings in the `[DNSServer]` section of the `ytuner.ini` file. I did this for my Yamahas. 😃

Yes, faster than vTuner. I used to pay for the service the last few years and it worked ok-ish (but slow). But now, the service from vTuner stopped to work for me even after I payed. This combined with the shutdown of FM radio in Switzerland was motivation enough to finally search for alternatives. I was surely impressed, that I had the solution (YTuner + Docker on my QNAP) running and properly configured in about 1 or 2 hours. This is obviously not always the case with open source solutions. So kudus to you for this.
Thanks for the hint about Wireshark and `InterceptDNS` even though I am not sure yet, if I will invest more time into this, since the solution works well enough for me.
 
> Yes, It does. 👍 See user manual page 26. Bookmarks are great, when you're browsing `Radiobrowser` and find a great music station you'd like to memorize with a just simple click(s) of the remote control... or remove from it. 👍

This I will definitively try.

> "Favorites" are stored on your QNAP. 👍 YTuner is native app build with FPC and that's why is pretty fast. 🚀 It seems you are a great Go developer 👏 so you know what I'm talking about. 😉

You are right. I am working (professionally and in my spare time) with Go for more than a decade by now and I know about the common roots of Go and Pascal. Interestingly, Pascal was the very first programming language that I learned beginning of the 90s. This was the time of Turbo Pascal and Borland (and of course MS DOS 😁).

> Please try `Radiobrowser` resources too... with one of high efficient cache options (see `RBCacheType` settings in the `[Radiobrowser]` section of the `ytuner.ini` file. 👍

I have configured the Radiobrowser and applied some filters (because without, it becomes barely usable).

This reminds me of an issue I faced, when I installed YTuner in Docker on my QNAP. I had an error in the logs (from Docker), that YTuner was not able to connect to https://www.radio-browser.info/ during startup. When I opened a shell inside of the running Docker instance, DNS lookup and accessing https://www.radio-browser.info/ worked perfectly fine. I have not invested a lot of time into this issue. I ended up with running ytuner during `docker build` on my notebook and basically adding the SQLite DB to the Docker container, such that on first run, a completely seeded DB is already present for YTuner.

Warm greetings,
Lucas

**Comment by vlad-6502** [comment]
Works on both Pioneer N-50A and Pioneer XC-HM81, running on Win64

The only strange behavior - station logo is not working correctly: 
- on N-50A looks like changing logo url isn't replacing the logo, old one is displayed
- on XC-HM81 logo isn't displayed at all

**Reply by coffeegreg** [reply]
Hi @vlad-6502 
Thank you for letting us know that YTuner works with Pioneer N-50A and Pioneer XC-HM81. :+1: 
I will add Pioneer N-50A and Pioneer XC-HM81 to the "Confirmed working" list.
> The only strange behavior - station logo is not working correctly:

Look at this issue : https://github.com/coffeegreg/YTuner/issues/47
I don't know what kind of format (JPG, PNG, others?) and size (width/height) of images works with Pioneer N-50A and Pioneer XC-HM81 and Pioneer N-30 too. Please try with some range of `IconSize` option of `ytuner.ini` file. YTuner try to on-the-fly convert and resize station logo image. Default is JPG format and 200x200px size ... but YTuner does not support absolutely all image formats such as `svg` or `webp`.
Test with different settings and station logo image file types and let me know. :+1: 
You can share your experiences here with @stokifan about Pioneer N-50A and N-30.

"6502" ... is this a good old one MOS CPU from Commodore VIC-20 or Apple I-III ? :wink:

**Reply by vlad-6502** [reply]
6502 - exactly! In name of CPU of the Soviet Apple II clone - "AGAT" ;)
When was a schoolboy spent long  hours programming in 6502 Assembler.

Will try to play on the weekend with logos, thank you!

**Reply by coffeegreg** [reply]
> 6502 - exactly! In name of CPU of the Soviet Apple II clone - "AGAT" ;)

Yeah... I knew it. :smile:  We are old boys ... :wink: 

> When was a schoolboy spent long hours programming in 6502 Assembler.

Me too .... LDA .... STA ... (MOS 6510) Do you still remember ? :wink: 
Assembler .... the best programming language in the world. FULL control over everything with HIGHEST performance.

**Comment by SuperMyron** [comment]
Hello, also working on Pioneer N70A.
Same avr.ini as Pioneer N50A.

**Reply by coffeegreg** [reply]
@SuperMyron 
Thanks for letting us know. :+1:  I'm going to add Pioneer N-70 A to the "Confirmed Working" list.

**Comment by J1So2** [comment]
Works fine with:
- Onkyo T-4070 (Network tuner)
- Onkyo TX-NR818 (AVR)
- NOXON NOVA II (A so called audio system - All in one device)

A big thank to you for a great piece of software!!!

**Reply by coffeegreg** [reply]
@J1So2 
Thanks for the info. :+1:  I'll add all the devices you mentioned to the "Confirmed Working" list. :rocket:

**Comment by Freleo** [comment]
Hello,

Thanks for sharing. 🙏

At home, it works with a Raspberry Pi 1 Model B+ and version 1.2.4 of ytuner on a **Pioneer N-50** and also on a **Pioneer PDX-Z9**.  👍


![Pioneer N-50](https://github.com/user-attachments/assets/a5b91257-700e-442c-9925-102de06b8782)

![Pioneer PDX-Z9](https://github.com/user-attachments/assets/284d3f6d-d9d3-48e2-a8f8-c9adccf001ec)

**Reply by coffeegreg** [reply]
Salut @Freleo :wink: 
Thanks for letting us know. Pioneer N-50 is already on the "Confirmed working" list but Pioneer PDX-Z9 is the new one and I'll add it to this list. 
I think I've seen these pictures before... :thinking: . Oh, yes :smiley: :
https://vintage-audio-heritage.fr/viewtopic.php?t=15745&hilit=ytuner&start=140
I think it was on this forum that I saw that someone tried to run YTuner binaries built for RPi2-4 `ytuner-1.2.6-armhf-linux-rpi-2-4.zip` on RPi1. But I think they ended up running the correct binaries from the file `ytuner-1.2.6-armhf-linux-rpi-0-1.zip` :wink: 
Unfortunately I don't follow this forum because I don't speak French and nothing stayed in my head from the two years of French studies at school, except "Je ne parle pas français." :smile: 
Greetings to all YTuner users from `vintage-audio-heritage.fr` forum. :+1: 

Why do you use YTuner v1.2.4 ? ... there is v1.2.6 in Releases.

**Reply by Freleo** [reply]
Hello,

> I think I've seen these pictures before...Oh, yes : https://vintage-audio-heritage.fr/viewtopic.php?t=15745&hilit=ytuner&start=140

Yep. I feel more at ease on the forums in the language of Molière. 😁


> I think it was on this forum that I saw that someone tried to run YTuner binaries built for RPi2-4 `ytuner-1.2.6-armhf-linux-rpi-2-4.zip` on RPi1. But I think they ended up running the correct binaries from the file `ytuner-1.2.6-armhf-linux-rpi-0-1.zip` 

That's me. But I have to admit that I'm not very good at it. 😜


> Greetings to all YTuner users from `vintage-audio-heritage.fr` forum. 

Message transmitted. 👍🥂


> Why do you use YTuner v1.2.4 ? ... there is v1.2.6 in Releases.

No deliberate intention, it's just that I based myself on discussion #41 
I'm going to start again with version 1.2.6, unless I wait for version 2.0.0. 😉

Have a good week-end. 😎

-- 
Freddy

**Reply by PapyRadio2022** [reply]
Hi,
I started to convince some people to use ytuner on the french forum [www.vintage-audio-heritage.fr](https://vintage-audio-heritage.fr/) (my pseudo there is jlp35). With some help, it was possible for non « geek » people were able to install and run ytuner with success. 
See https://vintage-audio-heritage.fr/viewtopic.php?t=15745 

Today I succeeded to get radio icons on the Pioneer N-50. 
You need to set
Iconsize=75
IconEndPointExtension=jpg
in ytuner.ini
And it works !

![obraz](https://github.com/user-attachments/assets/42d27bf2-ee6a-43de-9323-dd00442ac155)

many thanks for this excellent work. Our devices are back alive

**Reply by LbL-GH** [reply]
Thanks @PapyRadio2022 for identifying the proper settings for a Pioneer unit.  My AVR is an older model but the size now allows me to load the logo properly.  Merci

**Reply by coffeegreg** [reply]
> I started to convince some people to use ytuner on the french forum [www.vintage-audio-heritage.fr](https://vintage-audio-heritage.fr/) (my pseudo there is jlp35).

Hi @PapyRadio2022 ... :+1: **_vintage-audio-heritage_** is pretty cool but unfortunately I don't know language of Molière and sometimes google helps me understand what you're writing about :wink: 
That's great that you managed to guess what the correct size of the station logo will be for YTuner's on the fly image resizing for Pioneer N-50. For my Yamahas, 200px logo image size gave good results, so I set it as default in the `ytuner.ini` file, but of course each device needs its own settings. I'm glad that now we know what to set for Pioneer. :+1: 
Also remember what I wrote in Readme.md:
>YTuner can convert and resize on the fly logo image from JPEG, PNG, GIFF and TIFF (optionaly) to JPEG (default) or PNG format.

Please don't try with any other formats like `ico`, `svg` or `webp`. They won't work with current version of YTuner. I was writing about it here : https://github.com/coffeegreg/YTuner/issues/99#issuecomment-2704791079. Also remember that the format of the graphic file may sometimes be only apparent (look here : https://github.com/coffeegreg/YTuner/issues/99#issuecomment-2704832215).

>IconEndPointExtension=jpg

Are you sure about this?  Are you sure that your Pioneer needs this setting? As I wrote in `ytuner.ini` this parameter was suitable to resolve this issue : https://github.com/coffeegreg/YTuner/discussions/43. `IconEndPointExtension` doesn't mean graphics format of logo station image. This is only endpoint extension required in some special cases. Did you tried with `Iconsize=75` and with blank `IconEndPointExtension= ` ?

>With some help, it was possible for non « geek » people were able to install and run ytuner with success.

It would be great if someone on your forum was willing to write a short tutorial for the most beginners on how to start YTuner. It can also be your joint work that would show step by step for every beginner what and how to do it. For anyone who doesn't know what DNS, DHCP or even an IP address is. It would be good to describe from the beginning that YTuner should have a fixed IP address, how to separate such an address from the DHCP pool on the router, then all the AVR network settings on the example of Pioneer, add many images and photos and so on. You would be a great help to everyone who would also like to listen to music but has no idea about IT. Those who don't know anything about IT equipment especially need help.
You can make a pdf file or markdown file ([markdown basics](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)) and place it in this thread and I will make a link on the main page to this file with the description "_Tutorial for beginners by vintage-audio-heritage forum members_" (_Tutoriel pour débutants par les membres du forum vintage-audio-heritage_ :question: ) :wink: Of course in French and if you can manage it, other languages ​​are also welcome. :+1: 
The best idea is to create your own repository called, for example, 🤔` YTuner-doc` and put your doc files there and I will be happy to add a link to your repository of documents and anyone will be able to find and read them. 👍
What do you think about that? :fr: can help ? :wink:
In the meantime, I'm doing what I do best... coding new functionalities in YTuner :rocket: 

BTW... Have you tried the YTuner translator? How does it behave for the French language with following settings in `avr.ini` :
```
[Translator]
Enable=1
Auto=1
LanguageCode=fr
```
I would like to add a photo of your Pioneer with the station icon displayed to your message too. :+1:

**Reply by coffeegreg** [reply]
@TheBossME 
> Iconsize=75

Beatrice, I remember your Yamaha had a problem with displaying the station logo .. maybe you should experiment with different image sizes .. :thinking:

**Reply by TheBossME** [reply]
> @TheBossME 
> 
> > Iconsize=75
> 
> 
> 
> Beatrice, I remember your Yamaha had a problem with displaying the station logo .. maybe you should experiment with different image sizes .. :thinking: 

I'm not interested in that. The DSP-Z7 does not have a display only station Icons could be showed on TV. It works with Media streaming from my Synology NAS while Icons are stored to FLAC files and other formats.

In ytuner logfile there is no request for station images

**Reply by PapyRadio2022** [reply]
Thanks to have added the photo !
today I tried with
IconEndPointExtension=
and it does not work with N-50
So 
**IconEndPointExtension=jpg
Is absolutely needed**

I find somewhere a discussion about the N-50 unable to display images of albums when used as a music server if the size was not 75x75, this is why I tried this size. 
The idea to write a tuto for beginners is interesting.  This is what I did to help my friends in France. Lets think about it…

**Reply by coffeegreg** [reply]
@PapyRadio2022 
>IconEndPointExtension=jpg
Is absolutely needed

Good to know. :+1: I'll add a notice to the `ytuner.ini` file.

**Reply by coffeegreg** [reply]
@Freleo @PapyRadio2022 
https://vintage-audio-heritage.fr/viewtopic.php?t=15745&start=170#p278892
The messages like `Could not bind socket` or `Binding of socket failed` mean there is another process listening on this port (UDP53 for DNS service TCP80 for Web service). Another/old instance of YTuner still running? :wink: 

>Et tu n’aurais pas par hasard laissé le lancement automatique de ytuner dans le crontab ?

:+1:

**Reply by TheBossME** [reply]
Checkout with 
ss -tulpn

You will see used ports and by which application

**Reply by Freleo** [reply]
> @Freleo @PapyRadio2022 https://vintage-audio-heritage.fr/viewtopic.php?t=15745&start=170#p278892 The messages like `Could not bind socket` or `Binding of socket failed` mean there is another process listening on this port (UDP53 for DNS service TCP80 for Web service). Another/old instance of YTuner still running? 😉
> 
> > Et tu n’aurais pas par hasard laissé le lancement automatique de ytuner dans le crontab ?
> 
> 👍

You're absolutely right, gentlemen. 👍

Yesterday I installed version 1.2.6 of ytuner for my old Raspberry but I'd forgotten that I'd programmed the crontab to automatically launch version 1.2.4. 👎

And thanks to Papy Jean-Louis, I've also recovered the radio icons. 🙏

![DSC_0139](https://github.com/user-attachments/assets/8cb1c101-44ad-47ff-bc7e-fc1f952f20fb)

**Reply by coffeegreg** [reply]
@PapyRadio2022 @Freleo 
https://vintage-audio-heritage.fr/viewtopic.php?t=15745&sid=9704e410b429d6ede56d2f0dca658650&start=180#p279005
Databases from version v1.2.4 and version v1.2.6 are the same. The SQLite database will change in ... version 2.0.0. :eyes: 
If the data structure has been disrupted, then what you described may indeed help, although it should not happen, but ... Raspberry Pi 1 is very slow and therefore I advise against using a cache based on the SQLite database because it works slowly. `catMemStr` is the best option in this case. I also suggest leaving the option `RBCacheTTL=0` to prevent automatic refreshing of the whole database file because on such a slow device like RPi1 it may cause a lack of response for several minutes.

https://vintage-audio-heritage.fr/viewtopic.php?t=15745&sid=9704e410b429d6ede56d2f0dca658650&start=180#p279011
Guys, what are you drinking there? :beers: :wink:

**Reply by ChoroQ313** [reply]
Hello from Italy. May You please help me and suggest how to install Ytuner on my Pioneer PDX-Z9, which is connected via LAN to a Fastweb router and no longer finds web stations? I saw that it's possible, but I don't know how to do it. Thanks.

**Comment by ManuISEN** [comment]
Hi,
Works on Pioneer N-P01-K with a Raspberry Pi zero 2 W.
Thanks for your amazing work 😃

![IMG_3096](https://github.com/user-attachments/assets/49f0efab-1f74-46a4-a5d1-0cea4f2a671c)

**Reply by coffeegreg** [reply]
@ManuISEN 
Thanks for letting me know. I'm going to add Pioneer N-P01-K to the "Confirmed working" list :+1: 
YTuner v2.0.0 will be amazing ... still working on it. Hang on :rocket:

**Reply by TheBossME** [reply]
> YTuner v2.0.0 will be amazing ... still working on it. Hang on :rocket: 

@coffeegreg  What kind of improvements ? Ich bin wissbegierig 🧐

**Reply by coffeegreg** [reply]
You'll see. :wink: 
I still need time but I don't have much of it so the work is going slowly but still moving forward :+1: 
Of course, when I finish, I will ask everyone willing to take part in the tests. :rocket:

**Comment by phasperhoven** [comment]
Works with Harman Kardon HK3770. Server running on LXC Proxmox container. Thx!

**Reply by TheBossME** [reply]
#life without Proxmox is possible, but useless 🙋‍♀️🫶👍

**Reply by coffeegreg** [reply]
@phasperhoven 
Thanks for letting us know.  :+1: HK 3770 has been added to the "Confirmed working" list. :rocket:

**Comment by dzyndzla** [comment]
Works on Marantz M-CR610.
Thx for that awesome service. Cheers.

**Reply by coffeegreg** [reply]
@dzyndzla 
Thanks for letting me know. I'm going to add Marantz M-CR610 to the "Confirmed working" list. :rocket:

**Reply by dzyndzla** [reply]
@coffeegreg
Sure; I've noticed you made a typo there; there is '601' while it should be '610' instead ;-)

**Reply by coffeegreg** [reply]
@dzyndzla 
Yes, indeed ... typo :wink:

**Comment by PapyRadio2022** [comment]
Hi,
I don't know this model, so I am afraid I cannot help you much.


In any case you ned a computer (a raspberry is enough) to run Ytuner and to act as a DNS (Domain name Server) in your network to cheat the Pioneer reader. The system will then redirect the standard address of vtuner which is hardcoded in many devices to your local ytuner.

What you have to do is to change the DNS address on your Pioneer device. See "Setting up the network" in the user manual page 30.


Hope it helps

Jean-Louis


> Le 5 juil. 2026 à 16:51, ChoroQ313 ***@***.***> a écrit :
> 
> 
> Hello from Italy. May You please help me and suggest how to install Ytuner on my Pioneer PDX-Z9, which is connected via LAN to a Fastweb router and no longer finds web stations? I saw that it's possible, but I don't know how to do it. Thanks.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/coffeegreg/YTuner/discussions/23?email_source=notifications&email_token=A5XRLEIOE7A4QWIHZQDIN3T5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17538755>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A5XRLEPAZIHXZJ6ZU47NOFT5DJTQVAVCNFSNUABHKJSXA33TNF2G64TZHM3DGOJRGMZTANBRHNCGS43DOVZXG2LPNY5TMMJVG42DMNVBOYBA>.
> Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/A5XRLEMC2FYD73BV5A5J2QL5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ> and Android <https://github.com/notifications/mobile/android/A5XRLEJ5GRPT4K7WMJJMQND5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>. Download it today! 
> You are receiving this because you were mentioned.
> 

-- 
Prof. Jean-Louis Pazat
Membre de l'équipe Magellan, IRISA
responsable de la composante INSA-IRISA
www.insa-rennes.fr <http://www.insa-rennes.fr/>

￼

**Comment by ChoroQ313** [comment]
Ciao Jean-Louis.Thanks for your kindness answering me. I understood but I'm not able to find the ytuner.exe file that is necessary to install in Windows. I ask help as I saw that a person do it with a Pioneer PDX Z9 like mine. 
So I need to contact this person but I don't know how.Thanks again and excuse me.Daniele
    Il giorno lunedì 6 luglio 2026 10:59:17 CEST, PapyRadio2022 ***@***.***> ha scritto:  
 
 

Hi,
I don't know this model, so I am afraid I cannot help you much.


In any case you ned a computer (a raspberry is enough) to run Ytuner and to act as a DNS (Domain name Server) in your network to cheat the Pioneer reader. The system will then redirect the standard address of vtuner which is hardcoded in many devices to your local ytuner.

What you have to do is to change the DNS address on your Pioneer device. See "Setting up the network" in the user manual page 30.


Hope it helps

Jean-Louis


> Le 5 juil. 2026 à 16:51, ChoroQ313 ***@***.***> a écrit :
> 
> 
> Hello from Italy. May You please help me and suggest how to install Ytuner on my Pioneer PDX-Z9, which is connected via LAN to a Fastweb router and no longer finds web stations? I saw that it's possible, but I don't know how to do it. Thanks.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/coffeegreg/YTuner/discussions/23?email_source=notifications&email_token=A5XRLEIOE7A4QWIHZQDIN3T5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17538755>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A5XRLEPAZIHXZJ6ZU47NOFT5DJTQVAVCNFSNUABHKJSXA33TNF2G64TZHM3DGOJRGMZTANBRHNCGS43DOVZXG2LPNY5TMMJVG42DMNVBOYBA>.
> Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/A5XRLEMC2FYD73BV5A5J2QL5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ> and Android <https://github.com/notifications/mobile/android/A5XRLEJ5GRPT4K7WMJJMQND5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>. Download it today! 
> You are receiving this because you were mentioned.
> 

-- 
Prof. Jean-Louis Pazat
Membre de l'équipe Magellan, IRISA
responsable de la composante INSA-IRISA
www.insa-rennes.fr <http://www.insa-rennes.fr/>

￼


—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!

**Comment by PapyRadio2022** [comment]
Hi,
I am not sure there is a windows exe file. I use a raspberry raspbian system.
Jean-Louis

> Le 6 juil. 2026 à 11:13, ChoroQ313 ***@***.***> a écrit :
> 
> 
> Ciao Jean-Louis.Thanks for your kindness answering me. I understood but I'm not able to find the ytuner.exe file that is necessary to install in Windows. I ask help as I saw that a person do it with a Pioneer PDX Z9 like mine.  
> So I need to contact this person but I don't know how.Thanks again and excuse me.Daniele 
> Il giorno lunedì 6 luglio 2026 10:59:17 CEST, PapyRadio2022 ***@***.***> ha scritto: 
> 
> 
> 
> Hi, 
> I don't know this model, so I am afraid I cannot help you much. 
> 
> 
> In any case you ned a computer (a raspberry is enough) to run Ytuner and to act as a DNS (Domain name Server) in your network to cheat the Pioneer reader. The system will then redirect the standard address of vtuner which is hardcoded in many devices to your local ytuner. 
> 
> What you have to do is to change the DNS address on your Pioneer device. See "Setting up the network" in the user manual page 30. 
> 
> 
> Hope it helps 
> 
> Jean-Louis 
> 
> 
> > Le 5 juil. 2026 à 16:51, ChoroQ313 ***@***.***> a écrit : 
> > 
> > 
> > Hello from Italy. May You please help me and suggest how to install Ytuner on my Pioneer PDX-Z9, which is connected via LAN to a Fastweb router and no longer finds web stations? I saw that it's possible, but I don't know how to do it. Thanks. 
> > 
> > — 
> > Reply to this email directly, view it on GitHub <https://github.com/coffeegreg/YTuner/discussions/23?email_source=notifications&email_token=A5XRLEIOE7A4QWIHZQDIN3T5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17538755>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A5XRLEPAZIHXZJ6ZU47NOFT5DJTQVAVCNFSNUABHKJSXA33TNF2G64TZHM3DGOJRGMZTANBRHNCGS43DOVZXG2LPNY5TMMJVG42DMNVBOYBA>. 
> > Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/A5XRLEMC2FYD73BV5A5J2QL5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ> and Android <https://github.com/notifications/mobile/android/A5XRLEJ5GRPT4K7WMJJMQND5DJTQVA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGM4DONJVUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>. Download it today! 
> > You are receiving this because you were mentioned. 
> > 
> 
> -- 
> Prof. Jean-Louis Pazat 
> Membre de l'équipe Magellan, IRISA 
> responsable de la composante INSA-IRISA 
> www.insa-rennes.fr <http://www.insa-rennes.fr/> 
> 
> ￼ 
> 
> 
> — 
> Reply to this email directly, view it on GitHub, or unsubscribe. 
> Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today! 
> You are receiving this because you commented.Message ID: ***@***.***> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/coffeegreg/YTuner/discussions/23?email_source=notifications&email_token=A5XRLEN4CPWL644UGFYBEGT5DNUUJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ2TKMJXUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-17545517>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A5XRLELDEBGQVS2BMSTO5XT5DNUUJAVCNFSNUABHKJSXA33TNF2G64TZHM3DGOJRGMZTANBRHNCGS43DOVZXG2LPNY5TMMJVG42DMNVBOYBA>.
> Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS <https://github.com/notifications/mobile/ios/A5XRLEJP27OVTY6VZG2IUPL5DNUUJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ2TKMJXUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSVGM33PORSXEX3JN5ZQ> and Android <https://github.com/notifications/mobile/android/A5XRLEKJ2CGV4GDHBNVC4OD5DNUUJA5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNZVGQ2TKMJXUZZGKYLTN5XKO3LFNZ2GS33OUVSXMZLOOSXGM33PORSXEX3BNZSHE33JMQ>. Download it today! 
> You are receiving this because you were mentioned.
> 

-- 
Prof. Jean-Louis Pazat
Membre de l'équipe Magellan, IRISA
responsable de la composante INSA-IRISA
www.insa-rennes.fr <http://www.insa-rennes.fr/>

￼

**Comment by TheBossME** [comment]
checkout: https://github.com/coffeegreg/YTuner/discussions/108

Maybe the Server is still online..., not tested yet..

**Comment by gery357** [comment]
Hi,

YTuner works perfectly with legacy Naim Uniti 1 (software version 2.0.00, 24/96), YTuner is up and running successfully on a Raspberry Pi and Container QNAP. To make it work properly, two main things had to be configured: Main Menu Fix (The 4-line screen buffer issue): 
1) The Naim Uniti 1 screen displays exactly 4 lines of text. If avr.ini sends only 2 or 3 items in the menu, the Naim XML parser freezes—it displays the options but completely locks the remote control's up/down arrows. 
2) HTTPS-only streams still require manual fallback to HTTP URLs.

 Thank you so much for creating this lightweight and brilliant software!
